packages/lesswrong/components/icons/SettingsButton.tsx

```diff
@@ -33,15 +33,18 @@ const SettingsButton = ({classes, className, onClick, showIcon=true, label=""}:
   showIcon?: boolean
 }) => {
   if (label) {
-    return <span className={classes.iconWithLabelGroup} onClick={onClick}>
-      {showIcon && <Settings className={classNames(classes.icon, classes.iconWithLabel, className)}/>}
+    return <span className={classNames(classes.iconWithLabelGroup, className)} onClick={onClick}>
+      {showIcon && <Settings className={classNames(classes.icon, classes.iconWithLabel)}/>}
```

Comment:
Right, I remember now. This is safe because no one else was using the className prop.